docs/reference/errors-and-warnings/NU3004.md

```diff
@@ -29,3 +30,16 @@ The `require` validation mode does not support unsigned package and an unsigned
 ### Solution
 
 Please ensure that any package intended to be installed or passed to `nuget verify -signatures` command contains a package signature.
+
+## Scenario 2
+
+<pre>Package 'SamplePackage v1.0.0' from source 'https://contoso.com/index.json': This repository indicated that all its packages are repository signed; however, this package is unsigned.</pre>
+
+### Issue
+
+The repository indicated that all its packages are repository signed. However, an unsigned package was downloaded.
+
+### Solution
+
+This error may happen due to a bug when using multiple NuGet clients (see [NuGet/Home#7997](https://github.com/NuGet/Home/issues/7997)). Please upgrade to nuget.exe 5.1.0 or newer, Visual Studio 2019 16.1 or newer, and .NET SDK 2.1.70X or 2.2.30X or newer.
```

Comment:
I feel like we should call out that what teh issue description suggests to :)

Reply:
Good idea. Added a solution to contact the package source if this error actually happens.